src/asmcnc/job/job_data.py

```diff
@@ -405,3 +409,27 @@ def post_job_data_update_post_send(self):
         self.post_production_notes = ''
         self.batch_number = ''
         self.percent_thru_job = 0
+
+    def get_excluded_line_numbers(self, gcode):
```

Comment:
Can this function either be renamed or commented with details of what it's doing. Also, when does it run, as it's a for loop and has potential to lock up kivy

Reply:
renamed